doc/htmldoc/ref_material/glossary.rst

```diff
@@ -256,12 +288,6 @@ Glossary
  soma
    Cell body of the neuron.
 
- psp
-   Post-synaptic potential.
-
- PSC
-   Post-synatpic current.
-
  absolute refractory
    An interval after a neuron fires a spike to prevent it from firing a spike again.
```

Comment:
```suggestion
   Interval directly following a spike emission in which the sender neuron cannot fire again.
```

Reply:
The reason for the refractory period is of biological nature (depletion of neurotransmitters etc.). As the description was formulated it sounds as if there was a technical reason we would need to prevent follow-up spikes.